test/integration/command_test.exs

```diff
@@ -13,6 +13,11 @@ defmodule Integration.CommandTest do
     assert response["data"]["response"] == "req-opt response"
   end
 
+  test "running a command with a bad template", %{user: user} do
+    response = send_message(user, "@bot: operable:bad-template")
+    assert response["data"]["response"] == "@vanstee Whoops! An error occurred. There was an error rendering the template 'badtemplate' for the adapter 'test': %Protocol.UndefinedError{description: nil, protocol: String.Chars, value: %{\"bad\" => %{\"foo\" => \"bar\"}}}"
```

Comment:
This is undefined behavior as far as the mustache spec is concerned, since we're trying to interpolate a map. Ideally, we would just raise an error like `FuManchu.UnserializableContextError` with the value of context and some line number details and such. Does that sound reasonable?